core/models.py

```diff
@@ -55,17 +87,29 @@ def __str__(self):
 
 class College(models.Model):
     
+    """
+    This model is used to store the different colleges in the database.
+    Attributes:
+        name (str): The full name of the college.
+        address (str): The address of the college.
```

Comment:
Update the address in the docstring to country.

Reply:
Changed "address" to "country".